test/Core.Test/Services/StripePaymentServiceTests.cs

```diff
@@ -36,6 +36,22 @@ public async void PurchaseOrganizationAsync_Invalid(PaymentMethodType paymentMet
         Assert.Equal("Payment method is not supported at this time.", exception.Message);
     }
 
+    [Theory]
+    [BitAutoData(PaymentMethodType.BitPay)]
+    [BitAutoData(PaymentMethodType.BitPay)]
+    [BitAutoData(PaymentMethodType.Credit)]
+    [BitAutoData(PaymentMethodType.WireTransfer)]
+    [BitAutoData(PaymentMethodType.AppleInApp)]
+    [BitAutoData(PaymentMethodType.GoogleInApp)]
+    [BitAutoData(PaymentMethodType.Check)]
+    public async void PurchaseOrganizationAsync_SM_Invalid(PaymentMethodType paymentMethodType, SutProvider<StripePaymentService> sutProvider)
```

Comment:
This test is identical to the existing `PurchaseOrganizationAsync_Invalid` so we can just add the new parameters to that one.

Reply:
resolve